### PR TITLE
lov selection by id

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**
- #112 value_by_id
- #114 slider as a lov component
- fix doc for Gui.add_partial

### CODE REVIEW

File: gui/doc/lovComp.csv [hunk 1]
```diff
@@ -4,3 +4,4 @@ lov,"dict[str, any]",None,The list of values. See [GLOBAL-LOV] for details.
 adapter,Callable,`lambda x: str(x)`,"The function that transforms an element of <i>lov</i> into a <i>tuple(id:str, label:str|Icon)</i>."
 type,str,Type(lov-element)??,"Must be specified if <i>lov</i> contains a non-specific type of data (ex: dict).
 <i>value</i> must be of that type, <i>lov</i> must be an iterable on this type, and the adapter function will receive an object of this type."
+value_by_id,bool,False,Selection value is teh selected element of the LoV if False. It is the id of the selected element of LoV if True.
```
Comment (FabienLelaquais) [inline gui/doc/lovComp.csv]: "Selection value is teh selected element of the LoV if False. It is the id of the selected element of LoV if True"
->
"If False, the selection value (in _value_) is the selected element in _lov_. If set to True, then _value_ is set to the id of the selected element in _lov_.

**Review by FabienLelaquais — APPROVED** [review]
+++!

File: taipy/gui/utils/types.py [hunk 2]
```diff
@@ -61,7 +61,7 @@ def cast_value(self, value: t.Any):
                 warnings.warn(f"{self.get_name()}: Parsing {value} as float:\n{e}")
                 return 0.0
         else:
```
Comment (dinhlongviolin1) [inline taipy/gui/utils/types.py]: you dont need the else here

**Review by dinhlongviolin1 — APPROVED** [review]
Looks good to me

File: taipy/gui/utils/types.py (hunk 2 above)
Comment (FredLL-Avaiga) [reply taipy/gui/utils/types.py]: true

**Pull request merged**